### PR TITLE
{bio}[GCC/10.3.0] fumi_tools v0.18.2 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SethosII** [comment]
Test report by @SethosII
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ribnode012 - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2640 v2 @ 2.00GHz, Python 3.8.10
See https://gist.github.com/d9956aa89909dad47c8c4fcaa20535ff for a full test report.

File: easybuild/easyconfigs/f/fumi_tools/fumi_tools-0.18.2-GCC-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+builddependencies = [('CMake', '3.20.1')]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('pigz', '2.6'),
+    ('zlib', '1.2.11'),
+]
+
+separate_build_dir = True
+
```
Comment (branfosj) [inline easybuild/easyconfigs/f/fumi_tools/fumi_tools-0.18.2-GCC-10.3.0.eb:28]:
```suggestion
```

This is default for `CMakeMake`, so this can be removed.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/23fb8dacd94f3fcad23ebddf695d5a36 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @SethosII!

**Pull request merged**